src/index.html

```diff
@@ -9,6 +9,24 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="nav-item"><a href="#apple" class="nav-link is-active">Apple</a></li>
```

Comment:
```suggestion
          <li class="nav-item">
             <a href="#apple" class="nav-link is-active">Apple</a>
          </li>
```
format code in this way to make code more readable